dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -413,3 +416,47 @@ dashboard.header.updateTimestamp = function () {
     $('.project_updated_at').text("Not saved"); // TODO i18n
   }
 };
+
+function getSummarizedProgressForAnonymousUser () {
+  var summarizedProgress = {};
+  var levelProgress = {};
+  var cookie = document.cookie.split(';');
```

Comment:
I spent a little time looking at options and didn't use one because of Brent's reason. If we're going to do a lot more handling then I'll find something appropriate.